<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>510</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>510</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>471</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Program</string>
    </property>
    <widget class="QTableWidget" name="codeTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>464</width>
       <height>192</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>21</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
    <zorder>codeTable</zorder>
    <zorder>codeTable</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>471</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Data</string>
    </property>
    <widget class="QTableWidget" name="dataTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>464</width>
       <height>121</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>21</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>17</number>
     </attribute>
    </widget>
   </widget>
   <widget class="QComboBox" name="familyBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="partsBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Family:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Part name:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRead"/>
   <addaction name="actionProgram"/>
  </widget>
  <action name="actionRead">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../qrc/menu.qrc">
     <normaloff>:/icon/read.png</normaloff>:/icon/read.png</iconset>
   </property>
   <property name="text">
    <string>Read device</string>
   </property>
   <property name="toolTip">
    <string>Read device</string>
   </property>
  </action>
  <action name="actionProgram">
   <property name="icon">
    <iconset resource="../qrc/menu.qrc">
     <normaloff>:/icon/program.png</normaloff>:/icon/program.png</iconset>
   </property>
   <property name="text">
    <string>Program device</string>
   </property>
   <property name="toolTip">
    <string>Program device</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open (hex)</string>
   </property>
   <property name="toolTip">
    <string>Open </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qrc/menu.qrc"/>
 </resources>
 <connections/>
</ui>
